source/_footer.erb

```diff
@@ -0,0 +1,76 @@
+<div class="footer">
+  <div class="footer-container">
+
+    <div class="footer-container__tile">
+      <a class="footer-container__address-email" href="mailto:hello@unboxed.co">hello@unboxed.co</a>
+      <div class="footer-container__newsletter">
+        <p>Sign up to Unboxed updates</p>
+        <form action="//unboxedconsulting.us11.list-manage.com/subscribe/post?u=831facdc5caf30c092463f1c0&amp;id=e924413a0b" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" target="_blank" novalidate>
+          <div id="mc_embed_signup_scroll">
+            <input type="email" placeholder="your email address" value="" name="EMAIL" id="mce-EMAIL">
+            <div id="mce-responses">
+              <div id="mce-error-response" style="display:none"></div>
+              <div id="mce-success-response" style="display:none"></div>
+            </div>
+            <div style="position:absolute;left:-5000px;"><input type="text" name="b_831facdc5caf30c092463f1c0_e924413a0b" tabindex="-1" value=""></div>
+            <input type="submit" value="subscribe" name="subscribe" id="mc-embedded-subscribe">
+          </div>
+        </form>
+      </div>
+    </div>
+
+    <div class="footer-container__tile">
+      <div class="footer-container__address-content">
+        <p class="footer-container__address-city"><a href="http://goo.gl/maps/AuKrs" target="_blank"><i class="fa fa-map-marker footer__map-icon"></i>
+          London
+        <span>office</span></a></p>
+        <a class="footer-container__address-phone" href="tel:+442071834250"><i class="fa fa-phone footer__phone-icon"></i>
+          +44 20 7183 4250
+        </a>
+        <div class="footer-container__address-full">
+          <p>17 Blossom Street</p>
+          <p>London, E1 6PL</p>
+          <p>United Kingdom</p>
+        </div>
```

Comment:
It has just occurred to me that we could probably use an [`address`](http://www.w3schools.com/tags/tag_address.asp) tag here.

> The address element provides contact information for a document or part of a document. Information provided by address may include the names of the document’s maintainers, links to the maintainers’ Web pages, e-mail addresses for feedback, postal addresses, phone numbers, and so on. The address element is not appropriate for all postal and e-mail addresses; it should be reserved for providing such information about the contact people for the document.